tracetesting/definitions/test_create_with_id_exists.yml

```diff
@@ -1,5 +1,5 @@
-id: d85c3336-5048-4652-b564-63dfe2c83b14
-name: Test Create
+id: 7761457a-0a4b-4ada-8eab-d26c8ba48f90
```

Comment:
we had the same ID as the `Create Test` test, so we where overriding it each run